tests/utils.go

```diff
@@ -94,18 +94,6 @@ const (
 	DiskCustomHostPath     = "disk-custom-host-path"
 )
 
-func CreateSecret(name, namespace string, data map[string]string) {
-	virtCli := kubevirt.Client()
-
-	_, err := virtCli.CoreV1().Secrets(namespace).Create(context.Background(), &k8sv1.Secret{
-		ObjectMeta: metav1.ObjectMeta{Name: name},
-		StringData: data,
```

Comment:
It uses `StringData` and not `Data` like the new constructor.
Please reason why it is ok to change it.

Reply:
Done